network-api/networkapi/settings.py

```diff
@@ -92,9 +92,9 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = FILEBROWSER_DEBUG = env('DEBUG')
 
-# Force permanent redirects to the domain specified in TARGET_DOMAIN
-DOMAIN_REDIRECT_MIDDLWARE_ENABLED = env('DOMAIN_REDIRECT_MIDDLWARE_ENABLED')
-TARGET_DOMAIN = env('TARGET_DOMAIN')
+# Force permanent redirects to the domains specified in TARGET_DOMAINS
+DOMAIN_REDIRECT_MIDDLEWARE_ENABLED = env('DOMAIN_REDIRECT_MIDDLEWARE_ENABLED')
```

Comment:
is this missing a `DOMAIN_REDIRECT_MIDDLEWARE_ENABLED=(bool, False)` in the env block above? (would that obviate the need to specify it in env/confs?)

Reply:
yeah, I think this is what we need

Reply:
I'll add default env settings anyways.